modules/node-pool/main.tf

```diff
@@ -31,11 +31,18 @@ locals {
     propagate_at_launch = true
   }
 
-  cluster_autoscaler_tags = var.cluster_autoscaler ? {
-    key                 = "k8s.io/cluster-autoscaler/${data.aws_eks_cluster.this.id}"
-    value               = "owned"
-    propagate_at_launch = true
-  } : {}
+  cluster_autoscaler_tags = var.cluster_autoscaler ? [
+    {
+      key                 = "k8s.io/cluster-autoscaler/${data.aws_eks_cluster.this.id}"
+      value               = "owned"
+      propagate_at_launch = true
+    },
+    {
+      key                 = "k8s.io/cluster-autoscaler/enabled"
+      value               = "true"
+      propagate_at_launch = false
```

Comment:
set to `true`. We want to have this apply tags to the ASG _and_ the instances in it